packages/react-core/src/components/Slider/Slider.tsx

```diff
@@ -182,7 +194,11 @@ export const Slider: React.FunctionComponent<SliderProps> = ({
     e.stopPropagation();
     e.preventDefault();
 
-    diff = e.clientX - thumbRef.current.getBoundingClientRect().left;
+    if (isRTL) {
+      diff = thumbRef.current.getBoundingClientRect().left - e.clientX;
```

Comment:
I'm noticing a small bug when trying to drag the slider thumb via mouse and RTL enabled (but only for a continuous slider, where there's no "snapping" value). Maybe hard to tell from the video, but basically what I'm doing is clicking down and holding, and just *barely* moving my mouse; no matter what the thumb value jumps up by 2:


https://github.com/patternfly/patternfly-react/assets/70952936/baafb6d4-d031-4c6c-88b3-82346a307b9a

From what I can tell making the following update fixes it (a similar update would have to be made on line ~211 within the handleTouchStart):

```suggestion
      diff = thumbRef.current.getBoundingClientRect().right - e.clientX;
```